src/lib/make-snapshot.js

```diff
@@ -25,10 +25,9 @@ module.exports = async function makesSnapshot (options, browser) {
   const screenshotKey = baseKey + '.png'
   const htmlKey = baseKey + '.html'
   const content = getHtml(options.html, options.css, options.baseUrl)
+
   // this helps with debugging issues with snapshots.
-  // it is not waited for, so in some cases it might not complete the upload
-  // in time, but that is OK.
-  uploadToS3(htmlKey, content, 'text/html')
+  await uploadToS3(htmlKey, content, 'text/html')
```

Comment:
I've added `await`, as otherwise possible unhandled error coming from this function can kill the process and the handler won't try to make snapshot again. This usually takes 80-150ms, so it's a not a big cost I think. We could also stop sending it. I mostly look at logs and never at these files, but other devs might have a different workflow.